### PR TITLE
Chore adding platforms

### PR DESCRIPTION
**Pull request by Lloydyh**
This PR essentially adds all the necessary fields for data capture as well as some validation code

### CODE REVIEW

File: interface.html [hunk 1]
```diff
@@ -20,18 +20,459 @@
...
+        <label for="appFeaturedGraphic">Featured Graphic</label>
+      </div>
+      <div class="col-sm-7">
+        <input type="file" id="exampleInputFile" name="appFeaturedGraphic">
+        <p class="help-block">The feature graphic is displayed at the top of your store listing page in the Play Store app. Requirements: JPEG or 24-bit PNG (no alpha), Dimensions: 1024 px by 500 px. <a href="https://support.google.com/googleplay/android-developer/answer/1078870?hl=en-GB">Learn more</a></p></p>
+      </div>
+    </div>
+  </div>
+
+  <div class='appstore'>
```
Comment (squallstar) [inline interface.html]: double quotes

File: interface.html [hunk 2]
```diff
@@ -20,18 +20,459 @@
...
+      </div>
+    </div>
+  </div>
+
+  <div class="form-group clearfix appstore">
+    <div class="col-sm-4 control-label">
+      <label for="copyright">Contact Details</label>
+    </div>
+    <div class="col-sm-7">
+      <input type="text" name="firstname" class="form-control" id="firstname" placeholder="First name" value=""/>
```
Comment (squallstar) [inline interface.html]: should we prefix these names? like `contactFirstName`

File: js/formValidation.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+Fliplet.Env.set('appId', 1);
```
Comment (squallstar) [inline js/formValidation.js]: this needs to be removed

File: js/formValidation.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+var android;
```
Comment (squallstar) [inline js/formValidation.js]: android? give more context to this variables please :) I don't know what this means and what value could have.

File: js/formValidation.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+var requiredList;
+
+function validateForm(platform, submissionType){
+	return new Promise(function (resolve, reject){
+
+		requiredList = ['#appName',
```
Comment (squallstar) [inline js/formValidation.js]: indentation

File: js/formValidation.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
+Fliplet.Env.set('appId', 1);
+
+var android;
+var requiredList;
+
+function validateForm(platform, submissionType){
+	return new Promise(function (resolve, reject){
```
Comment (squallstar) [inline js/formValidation.js]: why does this return a promise?

File: js/formValidation.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+  					$(requiredList[input]).closest( '.form-group' ).addClass( 'has-error' );
+  				}
+			}
+		}
+
+	});  
+}
+
+// Fired from Fliplet Studio when the external save button is clicked
+Fliplet.Widget.onSaveRequest(function () {
```
Comment (squallstar) [inline js/formValidation.js]: why? isn't this on the other file too? I think these lines can be removed.

File: js/interface.js [hunk 8]
```diff
@@ -1,8 +1,9 @@
-// Fliplet.Env.set('appId', 6);
+Fliplet.Env.set('appId', 1);
```
Comment (squallstar) [inline js/interface.js]: this should be removed

File: interface.html (hunk 1 above)
Comment (Lloydyh) [inline interface.html]: :+1:

File: interface.html (hunk 2 above)
Comment (Lloydyh) [inline interface.html]: 👍

File: js/formValidation.js (hunk 3 above)
Comment (Lloydyh) [inline js/formValidation.js]: 👍

File: js/formValidation.js (hunk 4 above)
Comment (Lloydyh) [inline js/formValidation.js]: Removed

File: js/formValidation.js (hunk 6 above)
Comment (Lloydyh) [inline js/formValidation.js]: I suppose there is no need, will update

Comment (squallstar) [inline js/formValidation.js]: I mean, I'm fine if you want to return a promise and there's a reason for it :) but it does seem to be all sync, plus you're never resolving it.

File: js/formValidation.js (hunk 7 above)
Comment (Lloydyh) [inline js/formValidation.js]: Wasn't sure if this was needed, removed

File: js/interface.js (hunk 8 above)
Comment (Lloydyh) [inline js/interface.js]: removed

**Pull request merged**